pkg/api/common.go

```diff
@@ -0,0 +1,233 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// TODO: make this a separate package once we supports multiple internal versions
+// other than "".
+package api
+
+// Status is a return value for calls that don't return other objects.
+// TODO: this could go in apiserver, but I'm including it here so clients needn't
+// import both.
+type Status struct {
+	TypeMeta `json:",inline"`
```

Comment:
Are these types serialized?

By virtue of being in the api package, this is using api.TypeMeta and api.ListMeta, which isn't what we want.